templates/user/profile.html

```diff
@@ -5,76 +5,77 @@
 {% block description %}{{ user|capitalize }} is on Neomad with {{ articles.count() }} articles published and {% if user.allow_community %}sharing with the community{% else %}in hight privacy{% endif %}.{% endblock %}
 
 {% block extra_og %}
-<meta property=og:image content="{{ request.url_root[:-1] }}{{ user.avatar }}">
-<meta property=og:profile content={{ request.url }}>
-<meta property=og:profile:username content="{{ user.username }}">
-{% endblock %}
-
-{% block extra_css %}
-<link rel=stylesheet href={{ url_for('static', filename='css/profile.css') }}>
-<link rel=stylesheet href={{ url_for('static', filename='css/_article_list.css') }}>
+  <meta property=og:image content="{{ request.url_root[:-1] }}{{ user.avatar }}">
+  <meta property=og:profile content={{ request.url }}>
+  <meta property=og:profile:username content="{{ user.username }}">
 {% endblock %}
 
 {% block main %}
-<header>
-  <div class="profile-intro wrapper">
-    <figure>
-      <img class=avatar-img
-        name=avatar
-        src={{ user.avatar }}
-        width=100
-        height=100
-        onerror="this.src='{{ url_for('static', filename='images/avatar.png') }}'">
-    </figure>
-    <div>
-      <h1 name=username>{{ user.username }}</h1>
-      <p name=about>{{ user.about|htmlnewline }}</p>
-      {% if user.allow_community %}
-      <p>
-        {{ user }} is currently around <a href={{ url_for_trips(user) }} data-latlng={{ user.current_location and user.current_location|join(',') }}>the world</a>.</p>
-      {% else %}
-        <div class=private>
-          {{ user }} disabled community mode; the contents are therefore private.<br>
-          {% if user == current_user %}
-            <a href={{ url_for('privacy') }}>Change my community status</a>
-          {% endif %}
-        </div>
-      {% endif %}
-    </div>
-    <section class=social>
-      {% if user.socials.email -%}
-        <a class=email href=mailto:{{ user.socials.email }} title="Send an email">Send an email</a>
-      {%- endif %}
-      {% for name, url in user.socials.items() -%}
-        {% if name != 'email' and url -%}
-          <a class={{ name }} href={{ url }} target=_blank title="View {{ user }}'s profile on {{ name|capitalize }}">{{ name|capitalize }}</a>
+  <main>
```

Comment:
Why don't we use `<main>` in the base layout as we extend it always in the `block main`?

Reply:
because some of this main could get a special class to manage style and I prefer be obvious on this part, but maybe I'm wrong ^^

Reply:
No big deal.
A common technique is to set `<main class="{% block main_class %}{% endblock %}">`.
We ca do that later.

Reply:
Ok! I keep it in my mind 👍